sections/advanced/theming.md

```diff
@@ -43,6 +44,14 @@ render(
 );
 ```
 
+```
+// Example with object notation:
+const HighlightedSpan = Styled.Text(props => ({
```

Comment:
You should use `styled` instead of `Styled` to keep consistency with the rest of the documentation.
Also, what is `Text` in the context of this example? Perhaps it would be more fit to use a standard `span`?